gadget-container/gadgets/collector-process/main.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/link"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	PIN_PATH      = "/sys/fs/bpf/gadget"
+	BPF_ITER_NAME = "dump_task"
+)
+
+func increaseRlimit() error {
+	limit := &unix.Rlimit{
+		Cur: unix.RLIM_INFINITY,
+		Max: unix.RLIM_INFINITY,
+	}
+	return unix.Setrlimit(unix.RLIMIT_MEMLOCK, limit)
+}
+
+func main() {
+	if err := increaseRlimit(); err != nil {
+		log.Fatalf("failed to increase limit memlock limit: %s", err)
+	}
+
+	spec, err := ebpf.LoadCollectionSpecFromReader(bytes.NewReader(ebpfProg))
+	if err != nil {
+		log.Fatalf("loading asset: %s", err)
+	}
+
+	spec.Maps["containers"].Pinning = ebpf.PinByName
+
+	coll, err := ebpf.NewCollectionWithOptions(spec,
+		ebpf.CollectionOptions{
+			Maps: ebpf.MapOptions{
+				PinPath: PIN_PATH,
+			},
+		},
+	)
+	if err != nil {
+		log.Fatalf("creating eBPF collection: %s", err)
```

Comment:
These log.Fatalf calls should actually start in uppercase and include something like "Failed to" or "Error while" as this is going to be the beginning of the line, not going to get embedded into another error up the chain.

Reply:
Done